qiskit/qpy/__init__.py

```diff
@@ -170,6 +169,11 @@ def open(*args):
     it to QPY setting ``use_symengine=False``.  The resulting file can then be loaded by any later
     version of Qiskit.
 
+    With the removal of Pulse in Qiskit 2.0, QPY provides limited support for loading
+    payloads with pulse data. Loading a ``ScheduleBlock`` payload, a :class:`.QpyError` exception
+    will be raised. Loading a circuit with pulse gates, the circuit will contain custom
+    instructions without calibration data attached, leaving them undefined.
```

Comment:
I would make this a second note because it's logically different from the previous note about symengine.

```suggestion
.. note::

    Starting with Qiskit version 2.0.0, which removed the Pulse module from the library, QPY provides limited support for loading
    payloads that includes pulse data. Loading a ``ScheduleBlock`` payload, a :class:`.QpyError` exception
    will be raised. Loading a payload for a circuit that contained pulse gates, the output circuit will contain
    custom instructions **without** calibration data attached for each pulse gate, leaving them undefined.
```

Reply:
Got you, OK.